### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'extensionContainer'

### ISSUE THREAD

**Issue by ryanbugden** [ryanbugden/Eyeliner#11]
The extension misfires if the UFO is being opened concurrently with RF (RF is closed > Open UFO with... > RF). I think this is because the Glyph Window needs to be open before any "extension container" is recognized.

```
********************
Installing 'Eyeliner' report:
  Traceback (most recent call last):
      File "usher.py", line 24, in <module>
      File "/Applications/RoboFont_4p.app/Contents/Resources/lib/python3.7/mojo/subscriber.py", line 1415, in __init__
      File "/Users/Ryan/Library/Application Support/RoboFont/plugins/Eyeliner.roboFontExt/lib/eyelinerRF4.py", line 24, in build
    AttributeError: 'NoneType' object has no attribute 'extensionContainer'
```

**Comment by ryanbugden** [comment]
Difference b/w `build` and `started` 
https://robofont.com/documentation/reference/api/mojo/mojo-subscriber/#mojo.subscriber.build
https://robofont.com/documentation/reference/api/mojo/mojo-subscriber/#mojo.subscriber.started

**Comment by ryanbugden** [comment]
This is solved for the time being, by be6cbb3

Solved by assigning extension container only when upon the `glyphEditorDidOpen` callback.
